students/nicole/objects.js

```diff
@@ -0,0 +1,25 @@
+/* 
+   Objects Drills
+
+   Objects are used store relational data.
+
+   They will help us show how things are related to one another
+
+   Objects are created with {braces}
+*/
+
+
+const person = {
+    name: "Billy Bob",  // string
+    age: 96,  //number
+    job: null, // Null/ Undefined
+    friends: ["Dina, Paulina, Georgina"], //array
+    "his_real_address": "123 Fake Street", //possible but avoid
+}
+// Dot notation
+console.log(person.name);
+
+//  Bracket notation
+console.log(person{"name"});
```

Comment:
Bracket notation would look like this: `person['name']`. Brackets are the [square ones].

Braces are the {curly ones}